pages/api/search.js

```diff
@@ -0,0 +1,5 @@
+import getSubcategory from './s/[subcategoryId]'
+
+export default async function search(req, res) {
```

Comment:
Let's put it as it is in master
`
import Subcategory from './s/[subcategoryId]'

export default Subcategory
`